### ISSUE TITLE
Configuration file created by programing giving a critical error saying default is not valid

### ISSUE THREAD

**Issue by Ki11erRabbit** [pimutils/khal#1114]
After creating the configuration file using the built in configuration tool I get the error
``````
critical: in section [default] cal_local is not valid for 'default_calendar', must be one of ['default']
If your configuration file used to work, please have a look at the Changelog to see what changed.
``````
this is my configuration file
```
[calendars]

[[cal_local]]
path = ~/.calendars/*
type = discover

[locale]
timeformat = %I:%M %p
dateformat = %m/%d/%Y
longdateformat = %m/%d/%Y
datetimeformat = %m/%d/%Y %I:%M %p
longdatetimeformat = %m/%d/%Y %I:%M %p

[default]
default_calendar = cal_local
```
I have set up vdirsyncer correctly (unless I did it incorrectly for khal). Any suggestions for fixing my configuration file?

**Comment by geier** [comment]
Set `default_calendar = default` should fix the error, but this looks like a bug.

**Comment by Louson** [comment]
I confirm the issue. During the configuration, the program is asking:

```
Which calendar do you want as a default calendar?
(The default calendar is specified, when no calendar is specified.)
Configured calendars: gresille_calendar_local, gresille_contacts_local
Please type one of the above options (gresille_calendar_local, gresille_contacts_local) [gresille_calendar_local]: 
```

While gresille_caldendar_local and gresille_contacts_local are my vdirsyncer local pairs, not a calendar.

Here is my vdirsyncer conf:
```
# An example configuration for vdirsyncer.                                                               
#                                                                                                        
# Move it to ~/.vdirsyncer/config or ~/.config/vdirsyncer/config and edit it.                            
# Run `vdirsyncer --help` for CLI usage.                                                                 
#                                                                                                        
# Optional parameters are commented out.                                                                 
# This file doesn't document all available parameters, see                                               
# http://vdirsyncer.pimutils.org/ for the rest of them.                                                  

[general]
# A folder where vdirsyncer can store some metadata about each pair.                                     
status_path = "~/.local/share/vdirsyncer/status/"

# CARDDAV                                                                                                
[pair gresille_contacts]
# A `[pair <name>]` block defines two storages `a` and `b` that should be                                
# synchronized. The definition of these storages follows in `[storage <name>]`
# blocks. This is similar to accounts in OfflineIMAP.
a = "gresille_contacts_local"
b = "gresille_contacts_remote"

# Synchronize all collections that can be found.
# You need to run `vdirsyncer discover` if new calendars/addressbooks are added
# on the server.
collections = ["from a", "from b"]

# Synchronize the "display name" property into a local file (~/.contacts/displayname).
metadata = ["displayname"]

# To resolve a conflict the following values are possible:
#   `null` - abort when collisions occur (default)
#   `"a wins"` - assume a's items to be more up-to-date
#   `"b wins"` - assume b's items to be more up-to-date
conflict_resolution = "b wins"

[storage gresille_contacts_local]
# A storage references actual data on a remote server or on the local disk.
# Similar to repositories in OfflineIMAP.
type = "filesystem"
path = "~/.dav/contacts/"
fileext = ".vcf"

[storage gresille_contacts_remote]
type = "carddav"
url = "xxx"
username = "xxx"
# The password can also be fetched from the system password storage, netrc or a
# custom command. See http://vdirsyncer.pimutils.org/en/stable/keyring.html
password.fetch = "xxx"

# CALDAV
[pair gresille_calendar]
a = "gresille_calendar_local"
b = "gresille_calendar_remote"
collections = ["from a", "from b"]

# Calendars also have a color property
metadata = ["displayname", "color"]

conflict_resolution = "b wins"

[storage gresille_calendar_local]
type = "filesystem"
path = "~/.dav/calendars/"
fileext = ".ics"

[storage gresille_calendar_remote]
type = "caldav"
url = "xxx"
username = "xxx"
# The password can also be fetched from the system password storage, netrc or a
# custom command. See http://vdirsyncer.pimutils.org/en/stable/keyring.html
password = "xxx"
```

The vdirsyncer tree:
```
$ tree vdirsyncer
vdirsyncer
└── status
    ├── gresille_calendar
    │   ├── anniversaires.items
    │   ├── anniversaires.items-journal
    │   ├── app-generated--deck--board-26.items
    │   ├── app-generated--deck--board-26.items-journal
    │   ├── c.items
    │   ├── c.items-journal
    │   ├── les-loups.items
    │   ├── les-loups.items-journal
    │   ├── lucien.items
    │   ├── lucien.items-journal
    │   ├── personnel.items
    │   ├── personnel.items-journal
    │   ├── syndic.items
    │   ├── syndic.items-journal
    │   ├── syslinbit.items
    │   └── syslinbit.items-journal
    ├── gresille_calendar.collections
    ├── gresille_contacts
    │   ├── contacts.items
    │   ├── contacts.items-journal
    │   └── z-app-generated--contactsinteraction--recent.items
    └── gresille_contacts.collections
```

The remote is a nextcloud, the local is made from the synchronization.

**Comment by Louson** [comment]
Maybe it's fixed in #629
